wgsl/index.bs

```diff
@@ -3808,6 +3808,115 @@ If you want `vec4<f32>` you must provide 4 float values in the constructor.
 
 Issue: (dsinclair) Write out precedence rules. Matches c and glsl rules ....
 
+Issue: (mehmetoguzderin) Refine precedence rules ....
+
+<table class='data'>
+  <thead>
+    <tr>
+      <td>Precedence
+      <td>Operators
+      <td>Description
+      <td>Associativity
+  </thead>
+  <tr>
+    <td rowspan='1' style='text-align: left'><b>1 (highest)</b>
+    <td> <b><code>(</code></b><i>,</i> <b><code>)</code></b>
+    <td> Grouping
+    <td rowspan='1'> Left-to-right
+  <tr>
+    <td colspan='1' rowspan='4' style='text-align: left'>
+      <b>2</b>
+    <td> <b><code>!</code></b>
+    <td> Logical not
+    <td colspan='1' rowspan='4' style='text-align: left'>
+      Right-to-left
+  <tr>
+    <td> <b><code>~</code></b>
+    <td> Bitwise not
+  <tr>
+    <td> <b><code>+</code></b>
+    <td> Unary plus
+  <tr>
+    <td> <b><code>-</code></b>
```

Comment:
I believe this should be \`-\` instead of `<b><code>`

Reply:
Might be a problem with my setup but I chose this way of writing out to make it look better where there are multiple tokens in a single row

Reply:
I think we should stick with how bikeshed generates code blocks instead of injecting syntax.

Reply:
@dj2 I updated to stick with the bs.